osu.Game/Overlays/WikiOverlay.cs

```diff
@@ -82,6 +89,17 @@ protected void LoadDisplay(Drawable display)
             }, (cancellationToken = new CancellationTokenSource()).Token);
         }
 
+        protected override void UpdateAfterChildren()
+        {
+            base.UpdateAfterChildren();
+
+            if (articlePage != null)
+            {
+                articlePage.SidebarContainer.Height = DrawHeight;
+                articlePage.SidebarContainer.Y = Math.Clamp(ScrollFlow.Current - Header.DrawHeight, 0, Math.Max(ScrollFlow.ScrollContent.DrawHeight - DrawHeight - Header.DrawHeight, 0));
```

Comment:
What's this about? I can't say I'm following what this is trying to do very well.

Reply:
Hmm, honestly I just copied this calculation from `NewsOverlay`.

https://github.com/ppy/osu/blob/10acad6524d94335f710b5619fdee0bf39c5b64a/osu.Game/Overlays/NewsOverlay.cs#L133-L138

Reply:
I see. Seems that this is used to implement a sticky container. It would be nice to not have this logic duplicated in two containers, though.

Reply:
Yeah, something like abstract class for both overlays?

Reply:
Sounds like it could work, yes.

Reply:
I think that's out of scope in this PR for now.

Reply:
Not really, should be a pretty simple change. I would expose a protected virtual property or something to get the `SidebarContainer` if that's what you're worried about.

Reply:
I'm fine to proceed without it and do it as a follow-up, fwiw.

Reply:
Yeah, I don't want to refactor the whole overlay in this PR.